### ISSUE TITLE
'ETN group' filter

### ISSUE THREAD

**Issue by PieterjanVerhelst** [inbo/etn#114]
It would be interesting to allow a filter on ETN user groups in the functions `get_projects()`, `get_detections()`, `get_animals()`, `get_transmitters()` and `get_deployments`.
The majority of the groups are institutes like 'VLIZ', 'INBO' and 'UGENT' and would allow a filtering on their specific data and meta data. Even more, I am currently performing an eel meta-analysis using data from various institutes merged under the group 'eel-meta-analysis'. Hence, I would benefit from such a filter to download the data specifically added to this group.

**Comment by peterdesmet** [comment]
Sounds interesting, but the concept of a `group` is not available in any of the views. Where are those defined?

**Comment by PieterjanVerhelst** [comment]
@jreubens can you answer this question?

**Comment by jreubens** [comment]
I guess @PieterjanVerhelst is referring to owner_organisations.
Correct?

**Comment by PieterjanVerhelst** [comment]
that is correct 😃

**Comment by peterdesmet** [comment]
@PieterjanVerhelst so, the same field as discussed in #117? But extended to more views?

**Comment by PieterjanVerhelst** [comment]
indeed

**Comment by peterdesmet** [comment]
But it's a bit an odd concept to "own" deployments, animals and detections. And before we tackle this, I think we first need to figure out https://github.com/inbo/etn/issues/110#issuecomment-604893158

**Comment by PietrH** [comment]
Is this still something that needs to be added to the package?

**Comment by peterdesmet** [comment]
@PietrH I don't think so. With `get_*_projects(citation=TRUE)` one can now see the affiliation of the contact person. Knowing that, one can pass a selection of projects (animal or acoustic) to the functions to get relevant data.

@PieterjanVerhelst let me know if you agree and that we can close this issue.

**Comment by PieterjanVerhelst** [comment]
indeed, this is solved with the citations parameter in the function, so this can be closed.
